### PR TITLE
Add node affinity to monitoring-satellite

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

https://github.com/gitpod-io/gitpod/pull/5745 took a little longer than I expected to get merged. In the mean time I introduced a new external variable to monitoring satellite.

When rendering jsonnet code while not passing all external-variables, it fails with error:
```
RUNTIME ERROR: field does not exist: extvar 'node_affinity_label'
```

As @laushinka tried to deploy the stack for the first time after #5745, it failed 😬 

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

/werft withMonitoringSatellite

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **aledbf** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @aledbf` in a comment when ready.

Associated issue: *#5745*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by laushinka — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 723a35fc0d5487bb86f785660a89113b059472a4</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5860#pullrequestreview-763066869" title="Approved">laushinka</a>*

Associated issue: *#5745*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
